src/squlearn/kernel/matrix/kernel_matrix_base.py

```diff
@@ -214,7 +214,14 @@ def get_params(self, deep: bool = True) -> dict:
         Return:
             Dictionary with hyper-parameters and values.
         """
-        raise NotImplementedError()
+        # Create a dictionary of all public parameters
+        params = {}
+        params["feature_map"] = self._feature_map
+        params["executor"] = self._executor
```

Comment:
This is a bit cleaner
```suggestion
        params = {
            "feature_map": self._feature_map
            "executor": self._executor
        }
```